Automattic/newspack-newsletters
# Mailchimp: tags segmentation

### All Submissions:






### Changes proposed in this Pull Request:



Adds Mailchimp segmentation via tags. Tag creation will be handled in a separate PR. 

Closes #21

### How to test the changes in this Pull Request:

1. Create a newsletter, after selecting a list observe a new input below Groups selection – Tags
2. (Create some tags in Mailchimp if there are none)
3. Start typing a tag name, observe autocomplete in action, choose a tag
4. Observe the tag is added to the campaign on Mailchimp

### Other information:
